tools/ZAPDConfigs/MqDbg/SpecialEntranceList_OoTMqDbg.txt

```diff
@@ -0,0 +1,7 @@
+ENTR_RETURN_YOUSEI_IZUMI_YOKO
+ENTR_RETURN_SYATEKIJYOU
+ENTR_RETURN_2
+ENTR_RETURN_SHOP1
+ENTR_RETURN_4
+ENTR_RETURN_DAIYOUSEI_IZUMI
+ENTR_RETURN_GROTTO
```

Comment:
at least these have changed since the PR was opened

I forgot if the other entrances also did